build/setup.mk

```diff
@@ -25,6 +25,9 @@ HAS_SERVER ?= $(shell build/bin/manifest has_server)
 # Determine if a webapp is defined in the manifest.
 HAS_WEBAPP ?= $(shell build/bin/manifest has_webapp)
 
+# Determine if a /public folder is in use
+HAS_PUBLIC ?= $(wildcard server/public/*)
```

Comment:
Would you please change this to 
```suggestion
HAS_PUBLIC ?= $(wildcard public/.)
```

Reply:
Sure. The destination on the server would then just be the root of the plugin folder?

Reply:
If you are talking about the destination in the bundle: Yes.

Reply:
👍